components/modules/texts/text-02.tsx

```
@@ -4,68 +4,121 @@ import { cn } from "@/lib/utils";
-            <div className={cn("invisible text-4xl", className)}>{text}</div>
-        </motion.div>
-    );
-};
+  const baseTextStyle =
+    "absolute w-full text-4xl font-semibold tracking-tight bg-gradient-to-r from-slate-400 via-white to-slate-400 bg-clip-text text-transparent";
 
-export default TextSliced;
+  const getVariantStyles = (type: "top" | "bottom") => {
+    const baseOffset = type === "top" ? -splitSpacing : splitSpacing;
+
+    let transform: any = isVertical ? { y: baseOffset } : { x: baseOffset };
```

ESLint

[error] 'transform' is never reassigned. Use 'const' instead.

(prefer-const)